docs/public/static/components-gallery/base-theme.css

```diff
@@ -20,6 +20,84 @@
   --grey-700: #434d5b;
   --grey-800: #303740;
   --grey-900: #1c2025;
+
+  --background-surface-light-default: #fff;
+  --background-surface-light-soft: var(--grey-50);
+  --background-surface-light-hard: var(--grey-300);
+  --background-surface-light-highlight: var(--primary-500);
+  --background-surface-light-overlay: rgba(0,0,0,0.3);
+  --background-surface-dark-default: var(--grey-900);
+  --background-surface-dark-soft: var(--grey-800);
+  --background-surface-dark-hard: var(--grey-700);
+  --background-surface-dark-highlight: var(--primary-500);
+  --background-surface-dark-overlay: rgba(0,0,0,0.6);
```

Comment:
I don't know, it can work too, but I wonder about the developer experience when exporting and customizing this. 

If we define it directly on the component's vars, the user will have to tweak the tokens one by one to assure consistency--which will be challenging. In the current way, if the user wants to change surface levels, for instance, it'll be consistently applied everywhere just by changing  `--background-surface` tokens.